kube/src/service/auth.rs

```diff
@@ -188,4 +187,23 @@ mod tests {
             Error::Kubeconfig(ConfigError::InvalidBearerToken(_))
         ));
     }
+
+    fn test_token(token: String) -> RefreshableToken {
+        let expiry = Utc::now() + Duration::seconds(60 * 60);
+        let info = AuthInfo {
+            username: None,
+            password: None,
+            token: Some(token.clone()),
+            token_file: None,
+            client_certificate: None,
+            client_certificate_data: None,
+            client_key: None,
+            client_key_data: None,
+            impersonate: None,
+            impersonate_groups: None,
+            auth_provider: None,
+            exec: None,
+        };
```

Comment:
Possibly worth the `Default` derive for this :thinking:

Reply:
Done